src/web/routes/auth/integration.test.ts

```diff
@@ -70,41 +70,6 @@ describe('/auth', () => {
 				.expect({ type: '3LO', authorized: true });
 		});
 
-		it('should return a response indicating that user is not authorized if no credentials stored', async () => {
```

Comment:
**note:** Add new integration tests, reorder existing ones and resolve test flakiness (caused by a time-bound JWT token).